netbox/dcim/tests/test_cablepaths.py

```diff
@@ -1845,3 +2125,215 @@ def test_303_remove_termination_from_existing_cable(self):
             is_complete=True,
             is_active=True
         )
+
+    def test_401_non_symmetric_paths(self):
```

Comment:
These can probably be numbered from the 2XX series. But if we want to keep the 4XX numbering, we should add a note to the docstring for `CablePathTestCase`.